noir-projects/aztec-nr/aztec/src/messages/discovery/private_logs.nr

```diff
@@ -31,10 +31,10 @@ pub unconstrained fn fetch_and_process_private_tagged_logs<Env>(
     contract_address: AztecAddress,
     compute_note_hash_and_nullifier: ComputeNoteHashAndNullifier<Env>,
 ) {
-    // We will eventually perform log discovery via tagging here, but for now we simply call the `syncNotes` oracle.
+    // We will eventually perform log discovery via tagging here, but for now we simply call the `syncPrivateState` oracle.
     // This makes PXE synchronize tags, download logs and store the pending tagged logs in capsule array which are then
     // retrieved and processed here.
-    sync_notes(PENDING_TAGGED_LOG_ARRAY_BASE_SLOT);
+    sync_private_state(PENDING_TAGGED_LOG_ARRAY_BASE_SLOT);
```

Comment:
This change was wrong - this oracle is entirely unrelated (despite having the same name). What this oracle does is fetch tagged logs and place them in the capsule array so that the contract can later process them, it does not 'sync private state' and is not the same as the `sync_private_state` contract fn (though an exec of `sync_private_state` will at some point invoke this oracle).

Reply:
Are you fine with calling it sync_tagged_logs_and_remove_nullified_notes?
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/ceb00498-c8ed-4cb3-8dbc-372f882cbde3" />